deploy/template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: aws-account-shredder-template
+
+parameters:
+  - name: IMAGE
+    required: true
+    value: quay.io/agautam/aws-account-shredder
+  - name: IMAGE_TAG
+    value: latest
+
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: aws-account-shredder
+      labels:
+        name: aws-account-shredder
+    spec:
+      replicas: 1
```

Comment:
Could you expose the REPLICAS  as a parameter?

Reply:
I have added REPLICAS in parameter

Reply:
value : "1" . i have to use double quotes here. If i do not use double quotes , i get an error : 
    
 agautam  agautam-workstation  ../aws-account-shredder  oc process -f deploy/template.yaml 
error: failed to read input object (not a Template?): unable to decode "deploy/template.yaml": v1.Template.Parameters: []v1.Parameter: v1.Parameter.Value: ReadString: expects " or n, but found f, error found in #10 byte of ...|,"value":false}]}|..., bigger context ...|est"},{"name":"REPLICAS","required":true,"value":false}]}|...

it seems it is hitting this issue : https://github.com/kubernetes/kubernetes/issues/82296

Reply:
It is ok to add double quotes